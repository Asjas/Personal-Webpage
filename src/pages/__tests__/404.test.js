import React from 'react';
import { render } from 'react-testing-library';
import NotFoundPage from '../404';

const { debug } = render(<NotFoundPage />);
debug();
