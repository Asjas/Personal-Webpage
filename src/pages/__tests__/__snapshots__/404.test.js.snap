// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page matches snapshot 1`] = `
<div>
  <div
    class="sc-bdVaJa eMgOqm"
  >
    <h1
      class="sc-bwzfXH kfFPEG"
    >
      PAGE NOT FOUND
    </h1>
    <p
      class="sc-htpNat bUhMHN"
    >
      You just hit a route that doesn't exist... the sadness.
    </p>
    <a
      aria-current="page"
      class="sc-bxivhb OtERL"
      href="/"
    >
      Go Home
    </a>
  </div>
</div>
`;
